client/rpc/routes.go

```diff
@@ -0,0 +1,12 @@
+package rpc
```

Comment:
This should be merged with `client/rpc/root.go` as they host only one function each. Too tiny files to be both standalone.

Reply:
BTW `routes.go` seems definitely a better name than `root.go`